pkg/scheduler/cache/cache.go

```diff
@@ -157,6 +158,13 @@ type SchedulerCache struct {
 	// not be counted in pod pvc resource request and node.Allocatable, because the spec.drivers of csinode resource
 	// is always null, these provisioners usually are host path csi controllers like rancher.io/local-path and hostpath.csi.k8s.io.
 	IgnoredCSIProvisioners sets.Set[string]
+
+	multiSchedulerInfo
```

Comment:
Could we give more comment to this field so that everyone can better understand the context?

Reply:
Thanks for reminding~have added the background info.